src/index.html

```diff
@@ -1,14 +1,184 @@
+    </p>
+    <h2>Article 9</h2>
+    <p>
+      No one shall be subjected to arbitrary arrest, detention or exile.
+    </p>
+    <h2>Article 10</h2>
+    <p>
+      Everyone is entitled in full equality to a fair and public hearing by an independent
+      and impartial tribunal, in the determination of his rights and obligations and of any
+      criminal charge against him.
+    </p>
+    <h2>Article 11</h2>
+    <p>
+      <ol>
+        <li>Everyone charged with a penal offence has the right to be presumed
+            innocent until proved guilty according to law in a public trial at which he
+            has had all the guarantees necessary for his defence.
+        </li>
+        <li>No one shall be held guilty of any penal offence on account of any act or
+            omission which did not constitute a penal offence, under national or
+            international law, at the time when it was committed. Nor shall a heavier
+            penalty be imposed than the one that was applicable at the time the penal
+            offence was committed.
+        </li>
+      </ol>
+    </p>
+    <h2>Article 12</h2>
+    <p>
+      No one shall be subjected to arbitrary interference with his privacy, family, home
+      or correspondence, nor to attacks upon his honour and reputation. Everyone has
+      the right to the protection of the law against such interference or attacks.
+    </p>
+    <h2>Article 13</h2>
+    <p>
+      <ol>
+        <li>Everyone has the right to freedom of movement and residence within the
+            borders of each State.
+        </li>
+        <li>Everyone has the right to leave any country, including his own, and to
+            return to his country.
+        </li>
+      </ol>
+    </p>
+    <h2>Article 14</h2>
+    <p>
+      <ol>
+        <li>Everyone has the right to seek and to enjoy in other countries asylum from
+            persecution.
+        </li>
+        <li>This right may not be invoked in the case of prosecutions genuinely
+            arising from non-political crimes or from acts contrary to the purposes and
+            principles of the United Nations.
+        </li>
+      </ol>
+    </p>
+    <h2>Article 15</h2>
+    <p>
+      <ol>
+        <li>
+          Everyone has the right to a nationality.
+        </li>
+        <li>No one shall be arbitrarily deprived of his nationality nor denied the right to
+            change his nationality.
+        </li>
+      </ol>
+    </p>
+    <h2>Article 16</h2>
+    <p>
+      <ol>
+        <li>Men and women of full age, without any limitation due to race, nationality
+            or religion, have the right to marry and to found a family. They are entitled
+            to equal rights as to marriage, during marriage and at its dissolution.
+        </li>
+        <li>Marriage shall be entered into only with the free and full consent of the
+            intending spouses.
+        </li>
```

Comment:
```suggestion
        <li>
            Men and women of full age, without any limitation due to race, nationality
            or religion, have the right to marry and to found a family. They are entitled
            to equal rights as to marriage, during marriage and at its dissolution.
        </li>
        <li>
            Marriage shall be entered into only with the free and full consent of the
            intending spouses.
        </li>
```
исправь везде подобную ошибку

Reply:
ок, готово!